user-guide/Troubleshooting/RemoteLogCollection.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: RemoteLogCollection
+---
+
+# Remote Log Collection
+
+## What is it?
+
+The Remote Log Collection feature will allow our experts to instantly gather data from all nodes in your system. The ultimate goal is to enable our them to gather the necessary information from your system without your intervention. With this functionality experts from Skyline can trigger the LogCollector tool remotely, so they can collect the logs and memory dumps they need for their investigation.
+
+Currently the packages generated by the LogCollector are being saved on the DataMiner System, but in a future increment these packages will automatically be transferred to a secure storage in the cloud. By doing this the experts can immediately access the packages without any intervention from our users.
+
+## Requirements
+
+Before this feature can be used your system needs to meet a few requirements.
+
+1. The DataMiner Cloud Services need to be enabled on your system. If this is not yet the case, take a look at the [guide](xref:Connecting_your_DataMiner_System_to_the_cloud) on how to enable the DataMiner Cloud Services for your agent.
+1. Make sure you have installed a Cloud Pack with version 2.8.0 or higher. You can always find the latest Cloud Pack on [DataMiner Dojo](https://community.dataminer.services/downloads/).
```

Comment:
Maybe also include the specific versions needed for the feature? 
A user can't easily find the version of cloudpack they have installed, but they can see the different versions of the dxm's in the admin app

Reply:
would it not make more sense to have one centralized place in the docs that shows all the versions included in a cloud pack?
IMO this is easier to track and since we do not release the Cloud Pack that often...

Reply:
Possibly we could add release notes for the cloud packs after all, mentioning the included DxM versions, and referring to the change log for details?

Reply:
let's already get this out as is, we will look in the domain to set an approach for the DxM versions in cloud packs and our release strategy